sw/device/rom_exts/manifest.md

```diff
@@ -129,9 +133,10 @@ Notes:
     This is used by any ROM_EXT image parsers to identify a ROM_EXT image.
 
 1.  **Image Signature** This is a RSA 3k signature of all the fields that follow
-    the signature. This a sequence of 32-bit values. There is no data in the
-    image which is not signed, except the signature itself and the ROM_EXT
-    manifest identifier.
+    the signature. This a sequence of 32-bit values. The signed area of the
```

Comment:
nit: s/This a sequence/This is a sequence/

Reply:
Thanks!